squirrel.frontier/src/main/java/org/dice_research/squirrel/frontier/impl/FrontierSenderToWebservice.java

```diff
@@ -41,7 +42,7 @@
     private final long startRunTime = System.currentTimeMillis();
     private WorkerGuard workerGuard;
     private UriQueue queue;
-    private KnownUriFilter knownUriFilter;
+    private UriFilterComposer relationalUriFilter;
```

Comment:
Why do you need to use the Composer here? I think the composer is an extension of the KnownUriFilter. In that case, this change is not necessary.

Reply:
This comment has not been answered :thinking: